docs/data/base/components/button/UnstyledButtonIntroduction/css/index.js

```diff
@@ -54,42 +56,62 @@ function Styles() {
 
   return (
     <style>{`
-  .IntroductionButton {
-    font-family: 'IBM Plex Sans', sans-serif;
-    font-weight: 600;
-    font-size: 0.875rem;
-    line-height: 1.5;
-    background-color: ${cyan[500]};
-    padding: 8px 16px;
-    border-radius: 8px;
-    color: white;
-    transition: all 150ms ease;
-    cursor: pointer;
-    border: 1px solid ${cyan[500]};
-    box-shadow: 0 2px 1px ${
-      isDarkMode ? 'rgba(0, 0, 0, 0.5)' : 'rgba(45, 45, 60, 0.2)'
+      .Button {
+        all: unset;
```

Comment:
I am curious what's the motivation for adding this to the demos. It would break the default presets people may have in their projects.

Reply:
This is just a quick way to reset the UA styles, rather than going through and declaring each reset styles one-by-one.

> It would break the default presets people may have in their projects.

Any styles they need would be declared below this `all: unset` style. imo, in a component-based architecture, people shouldn't rely on leaky/global CSS to style elements. All styles they need should be self-contained and encapsulated in the component, so you can drop it into any screen/environment and it just works.

Reply:
Makes sense to me. I mentioned this because it conflicts with other parts of the docs where we promote using the CssBaseline component, for e.g. in Material UI. We should be consistent with this across the products in my opinion.